### ISSUE TITLE
bug? (not really): inconsistent practices

### ISSUE THREAD

**Issue by Agent-E11** [justinhj/battery.nvim#34]
I have noticed that there are some inconsistencies in the style of code.

Modules:
Returning a table with key-value pairs for functions: [example](https://github.com/justinhj/battery.nvim/blob/aa2f8ca3bde8b71709772413dca7853fb4e6ff11/lua/battery/acpi.lua#L86-L88)
A module table `M` (with key-value pairs set at the end): [example](https://github.com/justinhj/battery.nvim/blob/aa2f8ca3bde8b71709772413dca7853fb4e6ff11/lua/battery/battery.lua#L216-L219)
A module table `M` (with key-value pairs set at value declarations): [example](https://github.com/justinhj/battery.nvim/blob/main/lua/battery/health.lua#L13)

Strings:
Double quoted strings: [example](https://github.com/justinhj/battery.nvim/blob/aa2f8ca3bde8b71709772413dca7853fb4e6ff11/lua/battery/acpi.lua) (whole file)
Single quoted strings: [example](https://github.com/justinhj/battery.nvim/blob/aa2f8ca3bde8b71709772413dca7853fb4e6ff11/lua/util/file.lua#L1)

(There might be a couple other things)

My suggestions would be:

Module table `M` with key-value pairs set at value declaration:
```lua
local M = {}
function M.my_func() -- I prefer this to `M.my_func = function() ... end`, but I don't mind the second one that much
  ...
end
return M
```
And double quoted strings (though, I care much less about this one, I just care that they are all the same)

This might be a slight OCD issue 😅, but I think it would be nice to have some more consistent practices.

**Comment by justinhj** [comment]
I'm a fan of consistency. I was learning lua as I made the plugin so the style is a bit all over the place as you noticed. I do use luacheck in the commit checks so will gradually clean things up and add checks as time goes on.

**Comment by justinhj** [comment]
Have addressed in #36 
It's a lot of minor changes but will merge once I've tested it a bit. Feel free to give the branch a try. `stylua` is the branch name.
